src/Benchmarks.ServerJob/Framework.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Benchmarks.ServerJob
+{
+    public enum AspNetCoreVersion
```

Comment:
Why be this specific? The version chosen when running Benchmarks is already controllable using `ServerJob.Sources` and changes to `dependencies.props` in the named branches.